Update index.md (#4716)
segmentio/segment-docs: src/connections/reverse-etl/index.md

4. Select the Reverse ETL source you want to connect the destination to.
5. Enter the **Destination name** and click **Create Destination**.
6. Enter the required information on the **Settings** tab of the destination.

> 7. Navigate to the destination settings tab and enable the destination. If the destination is disabled, then Segment won't be able to start sync.

Reviewer: @dineshraj-arunmozhi curious why you want to add in that you have to navigate to the settings tab? From step 6, you should still be on the settings tab

Author: Hey @stayseesong, We have customers writing to us stating their sync has not started. While investigating, we noticed every time it has to do with the enabling the destination. Additionally, a couple of customers had felt that is not clear enough for them.

> 7. Navigate to the destination settings tab and enable the destination. Note that, if the destination is disabled then Segment will not be able to start sync.

Reviewer: ```suggestion
7. Navigate to the destination settings tab and enable the destination. If the destination is disabled, then Segment won't be able to start sync.
```
